Update containerd task driver docs. (#9550)
hashicorp/nomad: website/pages/docs/drivers/external/containerd.mdx

}
```

- `host_dns` - (Optional) `true` (default) or `false` By default, a container
  launched using `containerd-driver` will use host `/etc/resolv.conf`. This is

>   similar to [`docker behavior`][]. However, if you don't want to use

Reviewer: ```suggestion
  similar to [Docker's behavior]. However, if you don't want to use
```

Author: Fixed.

  host DNS, you can turn off this flag by setting `host_dns=false`.

> - `seccomp` - (Optional) Enable default seccomp profile. List of [`allowed syscalls`][].

Reviewer: Sorry, looks like we missed a spot: https://nomad-i2tmn1oma.vercel.app/docs/drivers/external/containerd#seccomp

```suggestion
- `seccomp` - (Optional) Enable default seccomp profile. List of [allowed syscalls].
```

Author: Fixed.

Reviewer: Thanks!

- `seccomp_profile` - (Optional) Path to custom seccomp profile.
  `seccomp` must be set to `true` in order to use `seccomp_profile`.

>   The default `docker` seccomp profile found [`here`](https://github.com/moby/moby/blob/master/profiles/seccomp/default.json)

Reviewer: Don't need the backticks here.

```suggestion
  The default `docker` seccomp profile found in the [Moby repository]
```

Author: Fixed.

 $ sudo tar -C /opt/cni/bin -xzf cni-plugins.tgz
```

Also, ensure your Linux operating system distribution has been configured
to allow container traffic through the bridge network to be routed via iptables.
These tunables can be set as follows:

```hcl
 $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-arptables
 $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-ip6tables
 $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-iptables
```

> To preserve these settings on startup of a nomad client node, add a file

Reviewer: ```suggestion
To preserve these settings on startup of a Nomad client node, add a file
```

Author: Fixed.

including the following to `/etc/sysctl.d/` or remove the file your Linux
distribution puts in that directory.

```hcl
 net.bridge.bridge-nf-call-arptables = 1
 net.bridge.bridge-nf-call-ip6tables = 1
 net.bridge.bridge-nf-call-iptables = 1
```

## Port Forwarding

> nomad supports both `static` and `dynamic` port mapping.

Reviewer: ```suggestion
Nomad supports both `static` and `dynamic` port mapping.
```

Author: Fixed.

1. **Static ports**

Static port mapping can be added in the `network` stanza.

```hcl
network {
  mode = "bridge"
  port "lb" {
    static = 8889
    to     = 8889
  }
}
```
Here, `host` port `8889` is mapped to `container` port `8889`.<br/>
**NOTE:** static ports are usually not recommended, except for
`system` or specialized jobs like load balancers.

2. **Dynamic ports**

Dynamic port mapping is also enabled in the `network` stanza.

```hcl
network {
  mode = "bridge"
  port "http" {
    to = 8080
  }
}
```
Here, nomad will allocate a dynamic port on the `host` and that port
will be mapped to `8080` in the container.

> You can also read more about `network stanza` in the [`nomad official documentation`](https://www.nomadproject.io/docs/job-specification/network).

Reviewer: We don't need to put code blocks around `stanza`. Generally we don't want to inline links, but put them at the end of the page instead.

```suggestion
You can read more about configuring networking under the [`network`] stanza documentation.
```

Author: Fixed.

## Service discovery

Nomad schedules workloads of various types across a cluster of generic hosts.
Because of this, placement is not known in advance and you will need to use
service discovery to connect tasks to other services deployed across your cluster.
Nomad integrates with Consul to provide service discovery and monitoring.

> A [`service stanza`](https://www.nomadproject.io/docs/job-specification/service)

Reviewer: "stanza" doesn't need to be monospace, and usually we prefer the term "block" to "stanza".

```suggestion
A [`service`] block
```

Author: Fixed.

[plugin-options]: #plugin_options
[host-network]: #host_network
[`mount options`]: https://github.com/containerd/containerd/blob/9561d9389d3dd87ff6030bf1da4e705bbc024130/mount/mount_linux.go#L198-L222

> [`docker behavior`]: https://docs.docker.com/config/containers/container-networking/#dns-services

Reviewer: ```suggestion
[Docker's behavior]: https://docs.docker.com/config/containers/container-networking/#dns-services
```

Author: Fixed.

> [`allowed syscalls`]: https://github.com/containerd/containerd/blob/master/contrib/seccomp/seccomp_default.go#L51-L390

Reviewer: ```suggestion
[Moby repository]: https://github.com/moby/moby/blob/master/profiles/seccomp/default.json
[allowed syscalls]: https://github.com/containerd/containerd/blob/master/contrib/seccomp/seccomp_default.go#L51-L390
[`network`]: /docs/job-specification/network
[`service`]: /docs/job-specification/service
```

Author: Fixed.
